### PR TITLE
oss-browser

### PR DESCRIPTION
**Pull request by hanabi1224**

### CODE REVIEW

File: bucket/oss-browser.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+    "version": "1.9.5",
+    "description": "ossbrowser is a graphical management tool developed by Alibaba Cloud. It provides features similar to those of Windows Explorer. Using ossbrowser, you can view, upload, download, and manage objects with ease.",
```
Comment (Ash258) [inline bucket/oss-browser.json]:
```suggestion
    "description": "View, upload, download, and manage files and folders with ease.",
```

File: bucket/oss-browser.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    "homepage": "https://github.com/aliyun/oss-browser",
+    "license": "Apache License 2.0",
```
Comment (Ash258) [inline bucket/oss-browser.json]:
```suggestion
    "license": "Apache-2.0",
```

File: bucket/oss-browser.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    "url": "https://oss-attachment.cn-hangzhou.oss.aliyun-inc.com/oss-browser/1.9.5/oss-browser-win32-x64.zip",
+    "hash": "sha1:d68f1944864d14f1ff49282e1169898d9f993ec0",
+    "extract_dir": "oss-browser-win32-x64",
+    "bin": [
+        "oss-browser.exe"
+    ],
+    "checkver": {
+        "url": "https://github.com/aliyun/oss-browser/blob/master/all-releases.md",
```
Comment (Ash258) [inline bucket/oss-browser.json]:
```suggestion
        "url": "https://raw.githubusercontent.com/aliyun/oss-browser/master/all-releases.md",
```

File: bucket/oss-browser.json [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+        "regex": "https://oss-attachment.cn-hangzhou.oss.aliyun-inc.com/oss-browser/(\\w+))/oss-browser-win32-x64.zip"
```
Comment (Ash258) [inline bucket/oss-browser.json]:
```suggestion
        "regex": "([\\d.]+)/oss-browser-win32-x64\\.zip"
```

File: bucket/oss-browser.json [hunk 5]
```diff
@@ -0,0 +1,19 @@
+{
+    "version": "1.9.5",
+    "description": "ossbrowser is a graphical management tool developed by Alibaba Cloud. It provides features similar to those of Windows Explorer. Using ossbrowser, you can view, upload, download, and manage objects with ease.",
+    "homepage": "https://github.com/aliyun/oss-browser",
+    "license": "Apache License 2.0",
+    "url": "https://oss-attachment.cn-hangzhou.oss.aliyun-inc.com/oss-browser/1.9.5/oss-browser-win32-x64.zip",
+    "hash": "sha1:d68f1944864d14f1ff49282e1169898d9f993ec0",
```
Comment (Ash258) [inline bucket/oss-browser.json]: Why `sha1` when there is no hash extraction?

Comment (hanabi1224) [reply bucket/oss-browser.json]: Got schema error without sha1:

Comment (Ash258) [reply bucket/oss-browser.json]: Use sha256 hash

Comment (Ash258) [reply bucket/oss-browser.json]: As soon as you resolve all comments just run `.\bin\checkver.ps1 oss-browser -f` and hashes will be automatically calculated.

Comment (hanabi1224) [reply bucket/oss-browser.json]: Got it, thx

**Comment by hanabi1224** [comment]
@Ash258 Regarding architecture and shortcuts, can you show me some examples?

File: bucket/oss-browser.json [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+            "url": "https://oss-attachment.cn-hangzhou.oss.aliyun-inc.com/oss-browser/1.9.5/oss-browser-win32-ia32.zip",
+            "hash": "2d01ad6e8fb3fd0b8f5c338a58364dc366232b1cfd5b5924b54fd8f9cf55aafd",
+            "extract_dir": "oss-browser-win32-ia32"
+        }
+    },
+    "bin": "oss-browser.exe",
+    "shortcuts": [
+        [
+            "oss-browser.exe",
+            "oss-browser"
```
Comment (Ash258) [inline bucket/oss-browser.json]: On screenshot the main window contain this title:

```suggestion
            "OSS Browser"
```

**Comment by hanabi1224** [comment]
@Ash258 No it's like a aws d3 explorer for another cloud platform called Aliyun

**Comment by hanabi1224** [comment]
I would suggest use the full description in quickstart section from readme

image

**Comment by Ash258** [comment]
I will think about it and when I arrive home I will write.

**Comment by linsui** [comment]
How about "View, upload, download, and manage files and folders with ease on Aliyun."

File: bucket/oss-browser.json (hunk 1 above)
Comment (rasa) [reply bucket/oss-browser.json]: Remove "ossbrowser is" as it is implied and unneeded.

**Comment by Ash258** [comment]
`Aliyun platform file manager.`
`Aliyun cloud file manager.`

**Comment by hanabi1224** [comment]
@Ash258 Can you plz take another look, refined a little bit because oss is just one of the file providers in the cloud

**Comment by hanabi1224** [comment]
@Ash258 kindly ping

**Pull request merged**